src/app/common/magic/hooks/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './use-bitcoin-network.hooks';
```

Comment:
We haven't typically used an index file to export all, so you are kinda introducing a new pattern here. @kyranjamie thoughts on starting to do this?

Reply:
I'm not totally against it when there's logical grouping, I do this in the ledger feature for example

Reply:
@fbwoolf @kyranjamie apologies! I did it merely out of habit, but I'm okay either way. Please, lmk if I should remove or leave them.

Reply:
Appears to be your call, I hadn't seen it in our codebase so just wanted to check ...sounds like it has been used elsewhere. 👍